features/fixtures/maze_runner/Assets/Scripts/MobileScenarioRunner.cs

```diff
@@ -221,7 +221,7 @@ private Configuration PreapareConfigForScenario(string scenarioName)
                     device.Jailbroken = true;
                     device.CpuAbi = new string[] { "poo", "baar" };
                     device.Orientation = "Custom Orientation";
-                    device.Time = new DateTime(1985, 08, 21, 01, 01, 01);
+                    device.Time = new DateTimeOffset(1985, 08, 21, 01, 01, 01, new TimeSpan(0));
```

Comment:
Slightly more idiomatic but same result
```suggestion
                    device.Time = new DateTimeOffset(1985, 08, 21, 01, 01, 01, TimeSpan.Zero);
```